app/controllers/planning_applications_controller.rb

```diff
@@ -155,6 +190,43 @@ def decision_notice
 
 private
 
+  def planning_application_params
+    permitted_keys = %i[address_1
+                        address_2
+                        application_type
+                        applicant_first_name
+                        applicant_last_name
+                        applicant_phone
+                        applicant_email
+                        agent_first_name
+                        agent_last_name
+                        agent_phone
+                        agent_email
+                        county
+                        created_at(3i)
+                        created_at(2i)
+                        created_at(1i)
+                        description
+                        proposal_details
+                        payment_reference
+                        postcode
+                        town
+                        uprn
+                        ward
+                        work_status]
+    params.fetch(:planning_application, {}).permit permitted_keys
```

Comment:
why are we use `fetch(:planning_application)` instead of `require(:planning_application)` here? Is there any situation that `params[:planning_application]` wouldn't be set?

Reply:
I went with the same convention we have in document_params right now:

```document_params = params.fetch(:document, {}).permit(:archive_reason, :name, :archived_at,
                                                         :numbers, :publishable, :referenced_in_decision_notice, :file, tags: [])```
   and because we are not actually requiring them, they are all optional - so it seemed to make sense?

Reply:
changed it